tests/e2e/main.go

```diff
@@ -128,6 +128,12 @@ var stepChoices = map[string]StepChoice{
 		description: "consumer double signing tests",
 		testConfig:  DefaultTestConfig(),
 	},
+	"consumer-double-downtime": {
```

Comment:
@p-offtermatt is this the right way to add a new e2e test?

Reply:
LGTM. Optionally, add the test case in https://github.com/cosmos/interchain-security/blob/e78593d8b9a2a071fd6dae79d91eeb035796c21e/tests/e2e/main.go#238if you want it to be a default test case